PaintingTrivia.py

```diff
@@ -160,27 +162,34 @@ def guess_the_saint():
         if x:
             return x
         try:
-            chosen_painting = int(request.form['chosen_painting'])
+            print request.form['chosen_saint']
+            chosen_painting = int(request.form['chosen_saint'])
         except KeyError:
 
             chosen_painting = -1
 
         key = 'right_guesses' if chosen_painting == session[
-            'selected_painting_id'] else 'wrong_guesses'
+            'selected_saint_id'] else 'wrong_guesses'
         session[key] += 1
 
     # Select a painter who has at least a painting
-    stmt = exists().where(Painting.painter_id)
-    painting_list = Painting.query.filter(stmt).order_by(func.random()).limit(4).all()
-    selected_painting = random.choice(painting_list)
+    # stmt = exists().where(Saint.paintings.any(id==Painting.id))
+    # painting_list = Painting.query.filter(stmt).order_by(func.random()).limit(4).all()
+
+    saints_list = Saint.query.filter(Saint.paintings.any()).limit(4).all()
```

Comment:
could you print out the sql query to see what it produces? (i dont know what this statement produces in the end)